pages/app/index.tsx

```diff
@@ -80,6 +71,10 @@ function App() {
   );
 }
 
+// The VR class needs to be set before any React rendering takes occurs.
+const vrDisabled = window.location.hash.includes('visualRefresh=false');
```

Comment:
Maybe I'm missing something, but do we need to parse the hash instead of using the `visualRefresh` value from the `AppContext`?